index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script
+      src="https://kit.fontawesome.com/27e53b408d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio | Joseph Ogbole</title>
+</head>
+<body>
+    <div class="hero">
+        <header class="header bg-white">
+          <nav class="navbar flex">
+              <div class="navbar__logo">
+                <a href="#" class="navbar__link link fz-18 fw-700 lh-20">MY LOGO</a>
+              </div>
+              <div class="navbar__menu">
+                <ul class="navbar__menu__ul ul">
+                  <li class="navbar__menu__li">
+                    <a href="#" class="navbar__menu__link link">Portfolio</a>
+                  </li>
+                  <li class="navbar__menu__li">
+                    <a href="#" class="navbar__menu__link link">About</a>
+                  </li>
+                  <li class="navbar__menu__li">
+                    <a href="#" class="navbar__menu__link link">Contact</a>
+                  </li>
+                </ul>
+              </div>
+              <div class="navbar__hamb">
+                  <img src="/images/hamburguer.png" alt="hamburguer icon">
+              </div>
+            </nav>
+        </header>
+  
+        <section class="head flex">
+          <h1 class="head__title fz-40 fw-700 lh-52">I'm Joseph Glad to see you!</h1>
+          <p class="head__dsc fz-16 fw-400 lh-24">
+            I’m a software developer! I can help you build a product , feature or
+            website Look through some of my work and experience! If you like what
+            you see and have a project you need coded, don’t hestiate to contact
+            me.
+          </p>
+          <a href="#" class="head__link fz-16 fw-500 lh-24">My Projects</a>
```

Comment:
- [ ] Please change this text "My Projects" to "LET'S CONNECT". Because it's indicating the social media icons it will be appropriate to use this.

Reply:
Done!